common/app/views/support/HtmlCleaner.scala

```diff
@@ -631,7 +631,7 @@ case class AtomsCleaner(atoms: Option[Atoms], shouldFence: Boolean)(implicit val
         atomId <- Some(bodyElement.attr("data-atom-id"))
         atomData <- findAtom(atomId)
       } {
-        val html = views.html.fragments.atoms.atom(atomData, shouldFence).toString()
+        val html = views.html.fragments.atoms.atom(atomData, amp, shouldFence).toString()
```

Comment:
can't you have the `if (amp)` clause here, then have a separate amp-atom template?